lms/djangoapps/ccx/views.py

```diff
@@ -183,7 +183,21 @@ def create_ccx(request, course, ccx=None):
                 override_field_for_ccx(ccx, vertical, hidden, True)
 
     ccx_id = CCXLocator.from_course_locator(course.id, ccx.id)  # pylint: disable=no-member
+    ccx_course = get_course_by_id(ccx_id)
     url = reverse('ccx_coach_dashboard', kwargs={'course_id': ccx_id})
+
+    # Enroll the coach in the course
+    email_params = get_email_params(course, auto_enroll=True, course_key=ccx_id, display_name=ccx.display_name)
+    enroll_email(
+        course_id=ccx_id,
+        student_email=request.user.email,
+        auto_enroll=True,
+        email_students=True,
+        email_params=email_params,
+    )
+
+    # Grant the coach staff access to the course
+    allow_access(ccx_course, request.user, "staff")
```

Comment:
Hi @sarina, good question -- this is actually something that another enhancement to CCX depends on -- I'll ask @giocalitri to chime in on which functionality this supports.